### ISSUE TITLE
Prize of War bug "Dark Elf Ch2"

### ISSUE THREAD

**Issue by ibahalwan** [WarEmu/WarBugs#16938]
Adept Telrothnal is bugged out up on the tower. You can target him, but can't really do any attack on him, even ranged. Thanks

**Comment by Kaelang** [comment]
I can see him spawned fine. Please can you screenshot where the NPC is?

**Comment by Kaelang** [comment]
Updating as went to take a look on live: 
![image](https://user-images.githubusercontent.com/60117106/82741802-d2a08800-9d4d-11ea-9244-7581bd6d7681.png)

Will keep this open for someone to confirm if anything can be done to stop something like this happening.

Cheers.

**Comment by ibahalwan** [comment]
Yes that's exactly what's happening. Thank you for the screenshot.

**Comment by ibahalwan** [comment]
![Capture](https://user-images.githubusercontent.com/65517730/82767328-0d3f1900-9e51-11ea-8b8f-e9da3609de59.PNG)

I just checked on it again. It seems like he is not always stuck inside the tower. Sometimes he is just there on the right spot. I'm not sure how to reproduce the bug.

**Comment by Lheana** [comment]
Someone must have pulled him, and the NPC got stuck inside the tower. I killed him yesterday so that he correctly respawns.
